src/app/test/test.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { AngularFireDatabase } from '@angular/fire/database';
+
+@Injectable({
```

Comment:
Do we need this file? if yes, create services folder have all the services inside that folder as well as rename the service name